client/web/web.go

```diff
@@ -70,6 +73,11 @@ func NewServer(devMode bool, lc *tailscale.LocalClient) (s *Server, cleanup func
 	if s.devMode {
 		cleanup = s.startDevServer()
 		s.addProxyToDevServer()
+
+		// Create new handler for "/api" requests.
+		// And protect with gorilla csrf.
+		csrfProtect := csrf.Protect(csrfKey())
+		s.apiHandler = csrfProtect(&api{s: s})
```

Comment:
as noted above, we probably want this csrfProtect on more than just the API handler.  But I also think we can safely pull it out of the `devMode` check at this point.  It should basically be a noop on the existing static web client, and we're past the 1.48 release, so we'll be switching things over relatively soon anyway.

Reply:
Yeah we probably actually could widen this to protect basically the whole Server handler. By just wrapping the whole thing. Once we get rid of the existing static site, though, the only piece that won't be wrapped will be the dev server proxy. So maybe okay as-is.